src/app/components/auth/auth.component.html

```diff
@@ -16,31 +16,35 @@ <h1 class="mt-5">SnyPy</h1>
           <div class="card rounded-0 auth-content">
             <div class="card-header text-center">
               <h2 class="mb-3 mt-3">
-                <ng-container *ngIf="ACTIVE_STATE === STATE_LOGIN">Login</ng-container>
-                <ng-container *ngIf="ACTIVE_STATE === STATE_REGISTER">Register</ng-container>
-                <ng-container *ngIf="ACTIVE_STATE === STATE_PASSWORD_FORGOT">Password forgot</ng-container>
+                <ng-container *ngIf="(activeState$ | async) === STATE_LOGIN">Login</ng-container>
+                <ng-container *ngIf="(activeState$ | async) === STATE_REGISTER">Register</ng-container>
+                <ng-container *ngIf="(activeState$ | async) === STATE_PASSWORD_FORGOT">Password forgot</ng-container>
               </h2>
```

Comment:
Use the async pipe for `activeState$` only once in the h2 tag:
```
<h2 class="mb-3 mt-3" *ngIf="activeState$ | async as activeState">
         <ng-container *ngIf="activeState === STATE_LOGIN">Login</ng-container>
         <ng-container *ngIf="activeState === STATE_REGISTER">Register</ng-container>
         <ng-container *ngIf="activeState === STATE_PASSWORD_FORGOT">Password forgot</ng-container>
</h2>
```

Reply:
Or wrap the content from the auth component in a `ng-container` and use `*ngIf="activeState$ | async as activeState"` there for the whole component.

Reply:
fixed, I added the condition on the top level container.